pkg/kanister.go

```diff
@@ -58,6 +59,14 @@ func Register(f Func) error {
 	return nil
 }
 
+func RegisteredFunctions() map[string]struct{} {
```

Comment:
Wouldn't we need the versions too if multiple are registered?

Reply:
I don't think we need version. Because we are just trying to check if the function with that name is supported or not. do you think we would need version information here.

Reply:
Should be okay for now - we can add it later if required 👍🏼